src/tests/sphgtod_t.c

```diff
@@ -23,7 +23,7 @@
 #else
 # define NS_P_S 1000000000ULL
 # define NS_P_uS 1000ULL
-# define ALLOWABLE_DRIFT 9000ULL
+# define ALLOWABLE_DRIFT 13000ULL
```

Comment:
@tuliom  On debian server shared by frediz it is around 900000ns for both the tests.